utils/resultstable.go

```diff
@@ -255,13 +255,36 @@ func prepareVulnerabilities(vulnerabilities []services.Vulnerability, results *R
 
 func sortVulnerabilityOrViolationRows(rows []formats.VulnerabilityOrViolationRow) {
 	sort.Slice(rows, func(i, j int) bool {
-		if rows[i].SeverityNumValue != rows[j].SeverityNumValue {
+		switch {
+		case rows[i].SeverityNumValue != rows[j].SeverityNumValue:
 			return rows[i].SeverityNumValue > rows[j].SeverityNumValue
+		case rows[i].Applicable != rows[j].Applicable:
+			return convertApplicableToScore(rows[i].Applicable) > convertApplicableToScore(rows[j].Applicable)
+		case rows[i].JfrogResearchInformation != nil || rows[j].JfrogResearchInformation != nil:
+			ok, value := JfrogResearchInformationSort(rows[i].JfrogResearchInformation, rows[j].JfrogResearchInformation)
+			if ok {
+				return value
+			}
+			return rows[i].IssueId > rows[j].IssueId
+		default:
+			return rows[i].IssueId > rows[j].IssueId
 		}
```

Comment:
```suggestion
		if rows[i].SeverityNumValue != rows[j].SeverityNumValue {
			return rows[i].SeverityNumValue > rows[j].SeverityNumValue
		}
		if rows[i].Applicable != rows[j].Applicable {
		         return convertApplicableToScore(rows[i].Applicable) > convertApplicableToScore(rows[j].Applicable)
		}
		if rows[i].JfrogResearchInformation != nil || rows[j].JfrogResearchInformation != nil:
			ok, value := JfrogResearchInformationSort(rows[i].JfrogResearchInformation, rows[j].JfrogResearchInformation)
			if ok {
				return value
			}
		}
		return rows[i].IssueId > rows[j].IssueId
```
I would use `if` statements for this, `switch` statement is not very readable in case you want to see what checks are performed first.
Also removed duplication for `return rows[i].IssueId > rows[j].IssueId`